### ISSUE TITLE
Accessibility: aria-label wrong position

### ISSUE THREAD

**Issue by melenapozzi** [ng-select/ng-select#2557]
### Describe the bug

When the dropdown is open the ng-dropdown-panel element has an aria-label which should be placed in its inner element with role="listbox" to avoid having an error on Success Criterion:A 4.1.2 Name, Role, Value of wcag 2.2.

### Steps to Reproduce

Create a simple ng-select component open the dropdown and run an accessibility tool (for example ARC Toolkit)
 

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

-

### Expected behaviour

Remove from the ng-dropdown-panel element the aria-label attribute and place it on the inner element with class 'ng-dropdown-panel-items'

### What version of the library you see this issue?

14.7.0

### What version of Angular are you using?

Angular 19

**Comment by VadimSidorovColPal** [comment]
Agreed, need to migrate aria-label from ng-dropdown-panel to .ng-dropdown-panel-items 
from
<img width="389" alt="Image" src="https://github.com/user-attachments/assets/ecf23958-ff3c-477e-893d-bc56c28b5c0a" />
to
<img width="708" alt="Image" src="https://github.com/user-attachments/assets/e2e78088-cb91-4c9f-b916-70537f0b16a0" />
